### ISSUE TITLE
(Re-)Add stats for posts

### ISSUE THREAD

**Issue by Ceddicedced** [MinnDevelopment/fxreddit#101]
Hey there,  

Loving the project so far! I was thinking it would be cool to have some more stats displayed. Things like:  
- Upvotes
- Comments
- Post Time

I’m curious what you think of this addition.

**Comment by MinnDevelopment** [comment]
I didn't want to include them because they are fundamentally ephemeral data, and embeds are heavily cached. Which means the stats you see are almost always outdated, it's better to just open the link to see live stats.

**Comment by Ceddicedced** [comment]
Fair point. I just thought it would have a more vanilla feeling as it would be closer to the original reddit embed, as this also includes stats. And maybe it would better align with the other embed fix tools like:

![grafik](https://github.com/user-attachments/assets/0fe21c20-f57c-476f-85e8-78169f7534d7)
![grafik](https://github.com/user-attachments/assets/32c162ca-d3a7-4add-b42e-2a93dd92402a)

As these also include stats. But this is just a thought I got.
